src/fheroes2/heroes/heroes_dialog.cpp

```diff
@@ -356,14 +357,16 @@ int Heroes::OpenDialog( const bool readonly, const bool fade, const bool disable
         buttonDismiss.hide();
     }
     else {
-        fheroes2::addGradientShadow( fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 ), display, dst_pt, { -3, 5 } );
+        fheroes2::addGradientShadow( fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 ), display,
+                                     { dst_pt.x, dst_pt.y - fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 ).height() / 2 }, { -3, 5 } );
```

Comment:
The same comment as the previous one:
```suggestion
        const fheroes2::Sprite & dismissPressed = fheroes2::AGG::GetICN( ICN::BUTTON_VERTICAL_DISMISS, 0 );
        fheroes2::addGradientShadow( dismissPressed, display, { dst_pt.x, dst_pt.y - dismissPressed.height() / 2 }, { -3, 5 } );
```